tools/lkl/lib/nt-host.c

```diff
@@ -288,6 +288,88 @@ struct lkl_host_operations lkl_host_ops = {
 	.jmp_buf_longjmp = jmp_buf_longjmp,
 };
 
+#ifdef PEX86_64
```

Comment:
Agreed. I also wondered why that should be necessary. We should use the POSIX for simplicity in maintainability. (One can use both APIs under Cygwin and MSYS, though)